packages/wdio-reporter/reporter.d.ts

```diff
@@ -0,0 +1,68 @@
+declare namespace WDIOReporter {
+    type TestState = 'passed' | 'pending' | 'failed' | 'skipped';
+    type ErrorType = 'AssertionError' | 'Error';
+
+    class Reporter {
+        constructor (options: Options);
+
+        onRunnerStart(): void;
+        onBeforeCommand(): void;
+        onAfterCommand(): void;
+        onScreenshot(): void;
+        onSuiteStart(suite: Suite): void;
+        onHookStart(hook: Hook): void;
+        onHookEnd(hook: Hook): void;
+        onTestStart(test: Test): void;
+        onTestPass(test: Test): void;
+        onTestFail(test: Test): void;
+        onTestSkip(test: Test): void;
+        onTestEnd(test: Test): void;
+        onSuiteEnd(suite: Suite): void;
+        onRunnerEnd(): void;
+
+        isSynchronised: boolean;
+    }
+
+    interface Options {
+        configFile: string;
+        logFile: string;
+        logLevel: string;
+        stdout?: boolean;
+    }
+
+    interface Suite {
```

Comment:
Can we somehow reuse Suite, Test, Hook (or at least something) from Webdriverio types

Reply:
I don't think that we can do that. 
It's another package